SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPropertyReplication/SpatialTestPropertyReplicationSubobject.cpp

```diff
@@ -95,11 +89,11 @@ void ASpatialTestPropertyReplicationSubobject::PrepareTest()
 		},
 		nullptr,
 		[this](float DeltaTime) {
-			RequireEqual_Int(TestActor->ReplicatedSubActor->TestReplicatedProperty, 999,
+			RequireEqual_Int(TestActor->ReplicatedSubobject->TestReplicatedProperty, 999,
 							 TEXT("The ReplicatedIntProperty on subobject should equal 999."));
 			FinishStep();
 		},
-		5.0f);
+		500.0f);
```

Comment:
Was it intentional to change the test timeout from 5 seconds to 500 seconds?

Reply:
Right, some leftover debug code.